about.html

```diff
@@ -0,0 +1,146 @@
+        </p>
+        <p class="desktopHeadlineDetail">
+          The Global Summit brings together the community of experts, academics,
+          and activists who comprise the Creative Commons affiliate network in a
+          different country every two. This year, we hope to expand our
+          invitation list including organizations and individuals Lorem, ipsum
+          dolor sit amet consectetur adipisicing elit. Voluptatem a dolores
+          quisquam illo enim alias voluptate maiores aliquid ut, porro ad
+          architecto, quos libero laboriosam inventore dolore esse debitis neque
+          praesentium ullam. Deserunt natus ea quo amet beatae voluptas aperiam?
+          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Voluptatibus
+          voluptatum veniam recusandae voluptas tenetur quos ipsam harum fugit,
+          itaque a magni soluta consequatur ullam ab hic deleniti quidem?
+          Aliquid voluptates beatae ad nisi voluptas illo totam optio error enim
+          dolore.
+        </p>
+        <div class="contact" id="contact">
+          <p>
+            Please contact us via email for any further question about PE Global
+            Summit 2022!
+          </p>
+          <a href="#">PEpakistan@digital.pk</a>
+        </div>
+      </section>
+    </div>
+
+    <!-- logo section start here  -->
+
+    <section class="logo1" id="logo">
+      <h2 class="logo_title">Pakistan ECOMERCE<br> Summit 2022 Logo</h2>
+      <h2 class="logo_dtitle">PAKISTAN ECOMERCE Summit 2022 Logo</h2>
+      <div class="guidbar"></div>
+      <div class="logo_img">
+        <a href="#">
+          <img
+            src="img/logo.jpeg"
+            alt="logo" height="200px"
+        /></a>
+      </div>
+    </section>
+
+    <!-- photes section start here  -->
+    <section class="photos" id="photos">
+      <h3 class="photos_title">See the past PE Summits</h3>
+      <div class="guidbar"></div>
+      <p class="photos_details">
+        Take a look at the last two PE Global Summits which took place in Buenos
+        Aires and in warsaw.
+      </p>
+      <div class="images">
+        <img
+          src="img/about_2022.png"
+          alt="CC Global Summit 2013 in Buenos Aires" />
+        <img
+          src="img/about_2011.png"
+          alt="CC Global Summit 2011 in Warsaw" />
+      </div>
+    </section>
+
+    <!-- partner section start -->
+
+    <section class="partner" id="sponser">
+      <h1 class="partner_title">Partner</h1>
+      <div class="guidbar"></div>
+      <div class="partner_logo">
+        <img src="img/14-mozilla-logo.png" alt="Mozilla-logo" />
+        <img src="img/15-google-logo.png" alt="Google-logo" />
+        <img src="img/16-naver-logo.png" alt="Naver-logo" />
+        <img src="img/17-daumkakao-logo (1).png" alt="Daumkakao-logo" />
+        <img src="img/18-Airbnb-logo.png" alt="Airbnb-logo" />
+      </div>
+    </section>
```

Comment:
- [ ] This section on the desktop view is supposed to be hidden and not visible. It should however be visible when the display is shifted to the mobile view. Kindly hide it for the desktop to match the Behance design.